content/blog/2018-09-04-birds-taxo-traits.md

```diff
@@ -0,0 +1,559 @@
+---
+title: "What are these birds? Complement occurrence data with taxonomy and traits information"
+slug: birds-taxo-traits
+authors:
+  - name: Maëlle Salmon
+    url: https://masalmon.eu/
+date: 2018-09-04
+preface: The blog post series corresponds to the material for a talk Maëlle will give at the [Animal Movement Analysis summer school in Radolfzell, Germany on September the 12th](http://animove.org/animove-2019-evening-keynotes/), in a Max Planck Institute of Ornithology.
+tags:
+- auk
+- taxize
+- traits
+- taxonomy
+- rphylopic
+- gifski
+- birder
+output: 
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+Thanks to the [second post of the series where we obtained data from
+eBird](https://ropensci.org/blog/2018/08/21/birds-radolfzell/) we know
+what birds were observed in the county of Constance. Now, not all
+species’ names mean a lot to me, and even if they did, there are a lot
+of them. In this post, we shall use rOpenSci’s packages accessing
+taxonomy and trait data in order to summarize the birds’ population of
+the county: armed with scientific and common names of birds, we have
+access to plenty of open data!
+
+### Getting and filtering the occurrence data
+
+For more details about the following code, refer to the [previous post
+of the series](https://ropensci.org/blog/2018/08/21/birds-radolfzell/).
+We only add a step to keep only data for the most recent years.
```

Comment:
two "only"'s so maybe rephrase to:
We add one step to keep only data

Reply:
ok "The single difference is our adding a step to keep only data for the most recent years."